migrations/20180322114835_add_second_label_to_answer.js

```diff
@@ -0,0 +1,11 @@
+exports.up = knex => {
+  return knex.schema.table("Answers", table => {
+    table.string("secondaryLabel");
+  });
+};
+
+exports.down = knex => {
+  return knex.schema.table("Answers", table =>
+    table.dropColumn("secondaryLabel")
+  );
+};
```

Comment:
We need a migration to seed any `DateRange` `label` and `secondaryLabel` with "Period from" and "Period to", respectively